syntax = "proto3";

package hello;

// import "google/protobuf/timestamp.proto";
// import "google/protobuf/any.proto";

service HelloService {
  rpc SayHello(HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
  string name = 1;
}

message HelloResponse {
  string message = 1;
}
